### ISSUE TITLE
Suggestions for more bug fixes.

### ISSUE THREAD

**Issue by Kichura** [j-tai/TieFix#2]
### Suggestion details:

1. Is it possible to have the dimensions bug fixed so that worlds do not get wiped when no worlds are present/worlds are upgraded?

2. Is it possible to have the StackOverflow bug fixed when attempting to debug advancements?

3. Is it possible to have the maps fixed so that colors of them become faked if nulled?

### Additional infos:

I may be aware that these are not necessarily annoying but are helpful for fixing server-related bugs when using fabric as server software.

**Comment by j-tai** [comment]
Unfortunately, server fixes are outside the scope of this mod. This mod is client-side only.

I'm not familiar with any of the bugs you have listed, so I'm not sure if any of them can be fixed client-side. If you could provide links to the relevant reports on the [bug tracker](https://bugs.mojang.com/) that'd be great.

**Comment by Kichura** [comment]
[One is from a mod directly which has no bug tracker yet but has resolved the issues with nulled maps](https://github.com/TechStreetDev/Map-Crash-Fixer-Mod/blob/1.18/src/main/java/me/techstreet/mapfixer/Main.java)

[Then there is this mod which fixes advancements based of debugging](https://www.curseforge.com/minecraft/mc-mods/advancements-debug)

[And this one is an actual bug tracker which was never resolved but deals with dimensions](https://bugs.mojang.com/browse/MC-197860)

**Comment by j-tai** [comment]
Thanks. I will look into the first two. The dimensions bug seems to be server-side.

**Comment by j-tai** [comment]
Is the "nulled maps" bug still an issue in 1.18.1? It seems to be fixed in vanilla:

```java
    private static MapColor getUnchecked(int id) {
        MapColor mapColor = COLORS[id];
        return mapColor != null ? mapColor : CLEAR;
    }
```

The code checks if the map color is unknown, and uses a transparent color in that case.

**Comment by Kichura** [comment]
> Is the "nulled maps" bug still an issue in 1.18.1? It seems to be fixed in vanilla:
> 
> ```java
>     private static MapColor getUnchecked(int id) {
>         MapColor mapColor = COLORS[id];
>         return mapColor != null ? mapColor : CLEAR;
>     }
> ```
> 
> The code checks if the map color is unknown, and uses a transparent color in that case.

Seems to be fixed in 1.18.1 as explained in the maps github but in prior versions, this is a slight problem.

**Comment by j-tai** [comment]
Okay. TieFix only targets the latest version of Minecraft so the fix is not necessary.

The StackOverflow bug seems to be server-side as well (please correct me if I'm wrong). Since TieFix is client side only, these won't be added.
